pod/perldelta.pod

```diff
@@ -77,6 +77,33 @@ XXX For a release on a stable branch, this section aspires to be:
 
 XXX Any deprecated features, syntax, modules etc. should be listed here.
 
+=head2 Downgrading a C<use VERSION> statement to below v5.11
+
+Attempting to issue a second C<use VERSION> statement that requests a version
+below than C<v5.11> when an earlier statement that requested a version higher
+than C<v5.11> has already been seen, will now print a deprecation warning.
+
+For example:
+
+    use v5.14;
+    say "The say statement is permitted";
+    use v5.8;                               # This will print a warning
+    print "We must use print\n";
+
+This is because of an intended related change to the interaction between
+C<use VERSION> and C<use strict>. Code which wishes to mix versions in this
```

Comment:
This mentions a relation between VERSION/strict but doesn't say what it is. I suggest:
```
This is because of an intended related change to the interaction between C<use VERSION> and C<use strict>. If you specify a version >= 5.11, strict is enabled automatically for you. If you request a version <= 5.11, strict is disabled for you.